db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1537,7 +1537,7 @@ Status DBImpl::FlushMemTable(ColumnFamilyData* cfd,
     }
```

Comment:
Since we are going to use only one bool, both write threads should be entered unbatched if writes_stopped if false.

Reply:
What do you think about this comment?

Reply:
Yes, I have merge them into one variable.

Reply:
The comments says:
> Since we are going to use only one bool, both write threads should be entered unbatched if writes_stopped if false.

Meaning that at this lined we need to enter_unbatched for both memtables.

Reply:
I dont's understand. We have enter_unbatched for `write_thread_`  while calling `IngestExternalFiles`, which is before we called `SwitchMemtable`. Why we need to enter_unbatched again for `write_thread_` in `SwitchMemtable`?

Reply:
Not for the main write_thread_, but for the 2nd one: `nonmem_write_thread_`. This is because we tell `SwitchMemtable` that we have already entered it so we should keep the promise.

Reply:
Do means that I should move `EnterUnbatched` outside of `SwitchMemtable`?

Reply:
Yes

Reply:
No. actually. Because inside SwithcMemtable you would skip it if writes_stopped is true.
This conversation is moving too slowly. I can go ahead and apply the change.